database/Procedures/UpdateBathymetry.sql

```diff
@@ -26,6 +26,12 @@ BEGIN
     [Group]      INT         NOT NULL
   );
 
+  DROP TABLE IF EXISTS ##T4;
+  CREATE TABLE ##T4 (
+    [geom]       GEOMETRY    NOT NULL,
+    [Group]      INT         NOT NULL
+  );
+
```

Comment:
Should this file be included?

Reply:
I am confused. This change was made on the develop branch, which was then merged into this branch. I have no idea why it's showing up in the file changes.

Reply:
**Addendum:** See my comment at the bottom of this PR.